Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -69,7 +69,10 @@ DownloadWidget::DownloadWidget(const URL& url)
     m_browser_image->load_from_file("/res/graphics/download-animation.gif"sv);
```

Comment:
(Commenting here because GitHub can't properly deal with commitbody comments.)

You changed the PR title, but the failing lint is about the *commit* title, as can be seen here: https://github.com/SerenityOS/serenity/actions/runs/5052025300/jobs/9064463078?pr=18993#step:3:83

Furthermore, the "category" of a commit subject usually about the bigger component, and only under certain circumstances about the file itself. The reason is that there are thousands of files, so there are multiple candidates where we might have a "DownloadWidget".

Please consider something more like the following:

```
Browser: Mention only file and directory in download widget

This is because the "From:" URL regularly overflows the line, leading
to an unreadable mess.
```

Reply:
Furthermore: Please rebase instead of creating merge commits.

Reply:
Should be all set now! Thank you for all of your feedback and help!!

Reply:
I don't think this is quite resolved yet. The commit title and description are identical and don't really explain what we're changing here, or why. I can see from the diff that there are "changes to the download widget file and directory text". But what's missing is a short description of what and why.

Ben's earlier suggestion sounds reasonable to me, maybe something more like:

```
Browser: Mention only file and directory in download widget

This is because the "From:" URL regularly overflows the line, leading
to an unreadable mess. Make sure that the labels for the widget
don't wrap as well, as the widget is fixed height and width.
```

The PR is also missing a reference to the issue that it's fixing. Normally a `fixes #1234` or a `ref #1234` if it doesn't fully solve the issue in the PR description does the trick. You could reference the issue in the commit description, but then the issue gets spammed every time you force-push which is less than ideal.